src/test/java/org/llorllale/cactoos/matchers/IsTrueTest.java

```diff
@@ -64,34 +64,23 @@ public void matchNegative() {
         ).affirm();
     }
 
-    /**
-     * Matcher prints the actual value(s) properly in case of errors.
-     * The actual/expected section are using only when testing is failed and
-     *  we need to explain what exactly went wrong.
-     */
-    @Test
-    public void describeActualValues() {
-        final Description desc = new StringDescription();
-        new IsTrue().matchesSafely(false, desc);
-        new Assertion<>(
-            "describes the test arg",
-            desc.toString(),
-            new IsEqual<>("<false>")
-        ).affirm();
-    }
-
-    /**
-     * Matcher prints the expected value(s) properly.
-     * The user has the ability to specify the description for the function.
-     */
     @Test
-    public void describeExpectedValues() {
-        final Description desc = new StringDescription();
-        new IsTrue().describeTo(desc);
+    public void describeCorrectly() throws IOException {
```

Comment:
@victornoel `describesCorrectly`, to conform to [EO test naming convention](https://www.javacodegeeks.com/2014/09/typical-mistakes-in-java-code.html).